force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -164,15 +235,20 @@ public with sharing class FlowControlLoopsCollections {
         // Initialize the result String
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
+        String result = inputString;
+        for (Integer i = 1; i < repeatCount; i++) {
+            if ( i == repeatCount) {
+                result += ' ' + inputString;
+            } else {
+                result += '; ' + inputString;
+            }
```

Comment:
Good work check out the methods repeat and removeEnd from the String class. You can make life easier. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm